repokid/cli/repokid_cli.py

```diff
@@ -681,6 +683,150 @@ def cancel_scheduled_repo(account_number, dynamo_table, role_name=None, is_all=N
                 role.account))
 
 
+def _check_access_advisor_age(role, role_name, account_number, max_days_old):
+    """Ensure access advisor data has been recently refreshed."""
+    old_aa_data_services = []
+    for aa_service in role.aa_data:
```

Comment:
can this block be cleaned up using some variables for the various times/periods so that the arithmetic is easier to understand from a first look. 

ie:
```
last_updated_time = datetime.datetime.strptime(aa_service['lastUpdated'], '%a, %d %b %Y %H:%M:%S %Z')
current_time = datetime.datetime.now()
expiration_time = datetime.timedelta(days=max_days_old)

        if( last_updated_time < current_time - expiration_time):
                old_aa_data_services.append(aa_service['serviceName'])
```
Probably need to play with var names, I'm tired and not picking the correct words/context for the block, but you get the idea of what I'm trying to convey.

Reply:
This was originally inline for the `repo_role()` method.  I refactored it into it's own method thinking I would need this code for the main method that I am adding, but it turns out that I don't.  I'll move this back into `repo_role()`, but won't be otherwise fixing the naming.

Not in this PR, anyways.